### DISCUSSION TITLE
New Grafana plugin validation disallows access to local filesystem. What to do?

### DISCUSSION THREAD

**Discussion by fr-ser** [grafana/grafana-csv-datasource#236] [Q&A]
Hello,

I am the author of another plugin (SQLite) and I am having trouble publishing a new version of my plugin as a new check has been added to the Grafana plugin checker that throws the following error:

> Found 1 error(s):
>
>  * Problem: It is not permitted to access the file system.

I assume you will have similar issues with the CSV plugin. Do you have any idea how to work around that?

**Comment by dburton-influxdata** [comment]
I just encountered this issue as well and looking for a way to resolve the issue.
